### PR TITLE
Fixes Abductors round end text

### LINKED ISSUE #9232: Abductors round-end text is a huge mess
Unreadable. It lacks proper formatting.

mess

### PR DESCRIPTION
**Pull request by Ty-Omaha**
Fixes #9232 

I think?

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Please test this.

"I think" won't do.

**Comment by Plotron** [comment]
Seems like an obvious fix.

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 1]
```diff
@@ -302,17 +302,17 @@
 /datum/game_mode/proc/auto_declare_completion_abduction()
 	var/text = ""
 	if(abductors.len)
-		text += "<br><span class='big'><b>The abductors were:</b></span>"
+		text += "<br><span class='big'><b>The abductors were:</b></span><br>"
 		for(var/datum/mind/abductor_mind in abductors)
 			text += printplayer(abductor_mind)
 			text += printobjectives(abductor_mind)
```
Comment (Plotron) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: We may need an additional new line here. If you look at the screenshot of the issue, there's no whitespace between player status and objective printout.

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 2]
```diff
@@ -302,17 +302,17 @@
 /datum/game_mode/proc/auto_declare_completion_abduction()
 	var/text = ""
 	if(abductors.len)
-		text += "<br><span class='big'><b>The abductors were:</b></span>"
+		text += "<br><span class='big'><b>The abductors were:</b></span><br>"
 		for(var/datum/mind/abductor_mind in abductors)
 			text += printplayer(abductor_mind)
```
Comment (Plotron) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: We may need an additional new line here. If you look at the screenshot of the issue, there's no whitespace between player status and objective printout.

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 3]
```diff
@@ -302,17 +302,17 @@
...
 			text += printobjectives(abductor_mind)
-		text += "<br>"
+			text += "<br>"
 		if(abductees.len)
-			text += "<br><span class='big'><b>The abductees were:</b></span>"
+			text += "<br><span class='big'><b>The abductees were:</b></span><br>"
 			for(var/datum/mind/abductee_mind in abductees)
 				text += printplayer(abductee_mind)
```
Comment (Plotron) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: Ditto.

**Comment by Fox-McCloud** [comment]
Please actually test this and post a screenshot of the results.

**Comment by Fox-McCloud** [comment]
Going to give you a few days to actually test this and demonstrate that it's been fixed, otherwise, it's going to be closed.

**Comment by Ty-Omaha** [comment]
It won't let me start the game because of the minimum player limits and I'm too lazy to change it, someone else can pick this up if they want

**Comment by taukausanake** [comment]
I'll do this once I get back home in three hours. For some reason I can't join a server, even a local one, on my campus computer

**Pull request closed without merging**